x/emissions/keeper/msgserver/msg_server_worker_payload_test.go

```diff
@@ -137,7 +137,13 @@ func (s *MsgServerTestSuite) TestMsgInsertWorkerPayload() {
 	blockHeight := workerMsg.WorkerDataBundle.InferenceForecastsBundle.Forecast.BlockHeight
 
 	ctx = ctx.WithBlockHeight(blockHeight)
-	_, err := msgServer.InsertWorkerPayload(ctx, &workerMsg)
+
+	err := s.emissionsKeeper.AddToTopicWorkerWhitelist(ctx, topicId, workerMsg.WorkerDataBundle.Worker)
+	require.NoError(err)
+	err = s.emissionsKeeper.AddToTopicWorkerWhitelist(ctx, workerMsg.WorkerDataBundle.TopicId, workerMsg.WorkerDataBundle.Worker)
```

Comment:
These two lines seem redundant.

Reply:
yup redundant. think i was testing the test!